src/Sonarr.Api.V3/Update/UpdateController.cs

```diff
@@ -5,10 +5,12 @@
 using NzbDrone.Core.Update;
 using NzbDrone.Core.Update.History;
 using Sonarr.Http;
+using Sonarr.Http.REST.Filters;
 
 namespace Sonarr.Api.V3.Update
 {
     [V3ApiController]
+    [TypeFilter(typeof(LogDatabaseDisabledActionFilterAttribute<List<UpdateResource>>))]
```

Comment:
No need to prevent any updates from showing, just don't check for install dates in `GetRecentUpdates`